content/en/blog/_posts/2024-02-29-cri-o-seccomp-oci-artifacts.md

```diff
@@ -0,0 +1,488 @@
+---
+layout: blog
+title: "CRI-O: Applying seccomp profiles from OCI registries"
+date: 2024-02-29
```

Comment:
We should be OK to go for this date. There are some bits I want to get right around what we communicate - there are what felt to me like a few design snags.

See the overall comment in https://github.com/kubernetes/website/pull/45121#pullrequestreview-1890388459 to cover my take on this. TL;DR: we can describe snags or encourage a fix, but we ought to do at least one of those per snag.